config/rector.php

```diff
@@ -11,9 +16,39 @@
             __DIR__ . '/../tests',
         ]
     )
-    ->withPhpSets()
-    ->withRules(
-        [
-            // AddVoidReturnTypeWhereNoReturnRector::class,
-        ]
-    );
+    ->withPhpVersion(PhpVersion::PHP_73)
+    ->withPhpSets(
+        true
+    )
+    ->withSets([
+        // Rector sets
+
+        LevelSetList::UP_TO_PHP_73,
+        // LevelSetList::UP_TO_PHP_74,
+        // LevelSetList::UP_TO_PHP_80,
+        // LevelSetList::UP_TO_PHP_81,
+        // LevelSetList::UP_TO_PHP_82,
+        // LevelSetList::UP_TO_PHP_83,
```

Comment:
I don't think we need these as comments; it should be obvious by deduction what the next ones will be.

Reply:
Done.